roles/docker-garbage-collection/templates/backup-removal.sh.j2

```diff
@@ -29,8 +32,11 @@ for backup_file in $backup_images_path/*; do
             log "Removed $backup_file from backup"
         else
             log "Failed removing $backup_file from backup"
+            backup_removal_exit_code=1
```

Comment:
Is it any reason that you don't want to exit the script with a non-zero code immediately when you ran into this block?

Reply:
I did not exit immediately so that a failed removal does not prevent checking and removal of other backed up images. The exit code is saved so that it still fails overall.